src/blocks/backlight.rs

```diff
@@ -126,6 +129,27 @@ pub struct Config {
     ddcci_max_tries_write_read: u8,
 }
 
+pub async fn join_all_accept_single_ok<I, T>(iter: I) -> Result<Vec<T>>
```

Comment:
Probably should come up with a better name. It's a join all, but only results in an error if there is not at least 1 ok value.